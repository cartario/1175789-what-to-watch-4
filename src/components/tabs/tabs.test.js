import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs.jsx";
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore([]);
const store = mockStore({
  FILMS: {
    films: [
      {
        id: 1,
      },
    ],
  },

});

describe(`Tabs test`, () => {
  it(`should render tabs`, () => {
    const tree = renderer.create(
        <Provider store = {store}>
          <Tabs
            activeFilmId = {1}
          />
        </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
