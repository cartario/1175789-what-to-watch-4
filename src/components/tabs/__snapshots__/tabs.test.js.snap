// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs test should render tabs 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    />
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Awesome
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    <p />
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
      </strong>
    </p>
  </div>
</div>
`;
